### ISSUE TITLE
Developer Experience SIG Surveys

### ISSUE THREAD

**Issue by tsloughter** [open-telemetry/sig-end-user#58]
The Developer Experience SIG has kicked off and one of our goals is to conduct an end user survey to collect experiences of pain points devs experience that we can look to address as well as engage maintainers (not necessarily through a survey) on what features outside the spec for user convenience they have added or are thinking of adding to address common concerns. 

I'm opening this issue to start discussion on how we can work together and what your availability is.

**Comment by danielgblanco** [comment]
Hi @tsloughter, I assume this would be different to what's been discussed in https://github.com/open-telemetry/sig-end-user/issues/42 ?

**Comment by tsloughter** [comment]
@danielgblanco right, these are different. One is about contributors to the otel project and devex is about users of otel.

**Comment by tsloughter** [comment]
@danielgblanco could you link me to those google docs you shared in the last SIG meeting, of previous or being worked on surveys and questions that will need to be included.

**Comment by danielgblanco** [comment]
Sorry I missed this comment originally, but I believe this was discussed on Slack?
